frontend/scss/components/organisms/toc.scss

```diff
@@ -52,6 +52,7 @@
     height: auto;
     top: 90px;
     right: auto;
+    padding-bottom: 90px;
```

Comment:
nit: using a CSS var would've been nice hear to make it clear that both values are the same.